test/controllers/charge_controller_create_test.js

```diff
@@ -0,0 +1,199 @@
+'use strict'
+
+// Core dependencies
+const path = require('path')
+
+// NPM dependencies
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+const expect = require('chai').expect
+
+const chargeId = '42mdrsshtsk4chpeoifhlgf4lk'
+
+const card = {
+  'brand': 'visa',
+  'type': 'CREDIT',
+  'corporate': false,
+  'prepaid': 'NOT_PREPAID'
+}
+
+const chargeData = {
```

Comment:
I think we should use `payment_fixtures`#`validChargeDetails().getPlain()` to construct this JSON. Having the connector response structure in one place makes the test code easier to maintain and the fixtures are also used by pact tests so we have some confidence that the JSON structure they produce is valid.